.github/workflows/ci.yml

```diff
@@ -139,3 +139,51 @@ jobs:
       - name: Build
         run: |
           ./native/scripts/windows-build.bat
+  tests:
+    runs-on: ubuntu-latest
+    env:
+      IMG: local-tomcat
+      HTTPD_IMG: local-httpd
+      DEBUG: on
+      FOREVER_PAUSE: 30
+      ITERATION_COUNT: 2
+      TOMCAT_CYCLE_COUNT: 2
+    strategy:
+      fail-fast: false
+      matrix:
+        # Those are currently all tests we have. In CI only MODCLUSTER-{734,755}/testit.sh pass, so until we fix the issue, only these tests are enabled.
+        # testcase: [ basetests.sh, hangingtests.sh, maintests.sh, JBCS-1236/testit.sh, MODCLUSTER-640/testit.sh, MODCLUSTER-734/testit.sh, MODCLUSTER-755/testit.sh, MODCLUSTER-785/testit.sh ]
+        testcase: [ MODCLUSTER-734/testit.sh, MODCLUSTER-755/testit.sh ]
```

Comment:
I see the idea, but that's not going to work here well.... We need a single job with a constant name ('tests' was fine!) so that we can set it as a required check. Moreover, this wastes resources, since the setup is done per test case.

Please just move this in a `for` look above `run_test` and keep this as single execution.

Reply:
Ok. I liked it because it was running in parallel thus saving some time (and the setup is not so demanding compared to tests themselves) and results were better accessible.

I updated it as requested.

Reply:
Yeah, the parallelization is nice. We can revisit if we find this an issue, but I don't foresee that.